### DISCUSSION TITLE
When added alpine script, all button can only pressed once

### DISCUSSION THREAD

**Discussion by jonathanoeijoeng** [alpinejs/alpine#3107] [7. Bugs (archived)]
I add this script in head section <script defer src="https://unpkg.com/alpinejs@3.x.x/dist/cdn.min.js"></script>
then when I click a button then suddenly all button cannot be pressed anymore. Not only hand cursor changed to arrow but buttons cannot be pressed again. 
I did refresh the page but same behaviour comes again.
All back to normal when comment or delete that script.

I use laravel 9.24 and livewire 2.

https://user-images.githubusercontent.com/7046560/184522571-4d5cba05-813d-46b0-9f28-d347e05b925d.mov

**Comment by SimoTod** [comment]
It looks like your modal has an invisible overlay that doesn't get removed after you close your div. You can verify if you right click and select inspect element. Hard to say more without seeing the code.

**Comment by jonathanoeijoeng** [comment]
Hi. Thanks for the reply. 
I found some errors in console as attach
<img width="376" alt="unknown-1" src="https://user-images.githubusercontent.com/7046560/184528453-e0acd6a5-430e-43b8-bf7a-edb0b0497b81.png">

**Reply by SimoTod** [reply]
Unhide the warnings in console. It should  tell you what caused the error. Most of the time is a typo in one of your directives.

**Reply by jonathanoeijoeng** [reply]
actually those 4 are all error. I also attached screenshot where I put CDN link
![image](https://user-images.githubusercontent.com/7046560/184560276-d99ed971-efc2-46ac-8737-d6bf1dfc0af9.png)

**Reply by SimoTod** [reply]
What i meant is that Alpine logs a more detailed entry in the console (but as a warning) before the actual error trying to explain what went wrong. Your first screenshot says you have 2 hidden log entries so i was wondering if one of the two was that warning.

Are you saying that you are not even using alpine (so you don't have any html attribute with x-data) but just importing the script? It's unusual, the library is used by thousands of people so it's more likely  to be something in your settings.

**Reply by jonathanoeijoeng** [reply]
Hi, yes those are error. I have filtered out warning and info.
I search for all file and apparently there is 1 file has x-data which come from jetstream. 
I also tried type alpine in console without CDN script in head section but console already showed alpine version means I believe jetstream already load it.

**Reply by jonathanoeijoeng** [reply]
I guess, jetstream installed alpine via npm so it conflict when I add CDN script. 
I tried to add <h1 x-data="{ message: 'I ❤️ Alpine' }" x-text="message"></h1> and I already can see it in the browser
